x/concentrated-liquidity/pool_test.go

```diff
@@ -40,7 +40,7 @@ func (s *KeeperTestSuite) TestCalcOutAmtGivenIn() {
 	swapFee := sdk.ZeroDec()
 	priceLimit := sdk.NewDec(5004)
 	tokenIn, tokenOut, updatedTick, updatedLiquidity, err := s.App.ConcentratedLiquidityKeeper.CalcOutAmtGivenIn(s.Ctx, tokenIn, tokenOutDenom, swapFee, priceLimit, pool.Id)
-	expectedTokenIn := sdk.NewCoin("usdc", sdk.NewInt(0))
+	expectedTokenIn := sdk.NewCoin("usdc", sdk.NewInt(42))
 	expectedTokenOut := sdk.NewCoin("eth", sdk.NewInt(42))
```

Comment:
I am a little confused on why the expected in and out would be the same here

Reply:
The swap here is in 42 usdc, out 42 eth

Reply:
ser where can I find this deal lol

Reply:
ser no changes with the expected numbers of test cases

Reply:
The current tests also puts in 42 usdc, gets 42 eth out right?

Reply:
No, the current test puts in 42000000usdc (42usdc) and gets out 008396 eth (price of 5000usd per eth)